examples/golang/atlas-basic.go

```diff
@@ -0,0 +1,62 @@
+package golang
+
+import (
+	mdb "github.com/mongodb/awscdk-resources-mongodbatlas-go/awscdkresourcesmongodbatlas"
+)
+
+type CdkTestAppGoStackProps struct {
+	awscdk.StackProps
+}
+
+func NewCdkTestAppGoStack(scope constructs.Construct, id string, props *CdkTestAppGoStackProps) awscdk.Stack {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+	stack := awscdk.NewStack(scope, &id, &sprops)
+
+	regionConfig := &mdb.AdvancedRegionConfig{
+		AnalyticsSpecs: &mdb.Specs{
+			EbsVolumeType: jsii.String("STANDARD"),
+			InstanceSize:  jsii.String("M10"),
+			NodeCount:     jsii.Number(1),
+		},
+		ElectableSpecs: &mdb.Specs{
+			EbsVolumeType: jsii.String("STANDARD"),
+			InstanceSize:  jsii.String("M10"),
+			NodeCount:     jsii.Number(3),
+		},
+		Priority:   jsii.Number(7),
+		RegionName: jsii.String("US_EAST_1"),
+	}
+
+	regionConfigs := &[]*mdb.AdvancedRegionConfig{regionConfig}
+	replicationSpecs := &[]*mdb.AdvancedReplicationSpec{
+		{
+			AdvancedRegionConfigs: regionConfigs,
+			NumShards:             jsii.Number(1),
+		},
+	}
+
+	ipAccessListProps := mdb.IpAccessListProps{
+		AccessList: &[]*mdb.AccessListDefinition{
+			{
+				Comment:   jsii.String("Open comment"),
+				IpAddress: jsii.String("10.10.0.0/24"),
```

Comment:
[nit] Can we replace the value with a placeholder string here? I can see people just using the code directly with no changes.

Ideally, lets define them in a struct passed into the method, so users can easily define in a single place.

Reply:
See https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/examples/l1-resources/cluster.ts for an example

Reply:
updated examples